P/Packmol/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Packmol"
+version = v"20.3.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://leandro.iqm.unicamp.br/m3g/packmol/packmol.tar.gz", "21c249670e08dfed142658030ea58bc4473483a4b9da263abff13588f3c47f31")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd packmol/
+mkdir -p $prefix/shared/licenses
+cp LICENSE $prefix/shared/licenses
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+make
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_gfortran_versions(supported_platforms())
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("packmol", :packmol)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
```

Comment:
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=20507&view=logs&j=40d8ebc2-c0bb-5d17-343c-5891a54e11f0&t=4153e422-92fb-5aba-a5fa-cc269bd69ce5&l=478
```
┌ Warning: To ensure that the correct version of libgfortran is found at runtime, add the following entry to the list of dependencies of this builder
│ 
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/wohhx/src/auditor/compiler_abi.jl:4
```
```suggestion
dependencies = [
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
```